### DISCUSSION TITLE
Please run as root.

### DISCUSSION THREAD

**Discussion by xixixiba** [oldkingcone/slopShell#8] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/95956613/148167386-124cfb79-8b3f-4e73-8cdc-c99b0d005372.png)
![image](https://user-images.githubusercontent.com/95956613/148167423-02e4921b-936e-4caf-81e0-3a95324d1779.png)
The root user is still prompted to run as the root user???

**Comment by oldkingcone** [comment, marked as answer]
You need to invoke the script with the sudo command.

**Reply by xixixiba** [reply]
![image](https://user-images.githubusercontent.com/95956613/148490470-894f941a-3c41-4fc0-8d1d-766ae5b02115.png)
Still not

**Comment by oldkingcone** [comment]
weird. ill make fixes and add it later. keep an eye out for a new push.

**Reply by xixixiba** [reply]
I use Kali. Is it Kali's problem???

**Reply by oldkingcone** [reply]
not sure yet, ive tested on debian based distro's and RPM distros, worked on both. Might just be a local issue, but ive disabled it for now. Just make sure you are invoking the script with sudo in order for the functions to run properly.

**Reply by xixixiba** [reply]
Thanks for your help

**Reply by oldkingcone** [reply]
no problem! if you have anymore issues, feel free to reach out.

**Reply by xixixiba** [reply]
I'll wait for subsequent updates before using it

**Reply by oldkingcone** [reply]
should be good to clone now. made a few alterations.

**Reply by xixixiba** [reply]
Everything ok  Thank you, man

**Reply by oldkingcone** [reply]
no problem!

**Comment by Pure-cai** [comment]
Execute ./setup.sh ,I have installed postgresql ，Line 54 Password required，Where is this password?

**Reply by oldkingcone** [reply]
can you open a new issue and provide screenshots please.
